modules/updates-and-upgrades.adoc

```diff
@@ -11,5 +11,4 @@ The decision regarding the need for a Service update to the Central instance and
 Customers have no control over when a Central service update occurs. For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
 Upgrades to the version of {rh-rhacscs-first} are considered part of the service update.
 
-The version of {product-title-managed-short} used on Secured Clusters must match the version of the Central instance of {product-title-managed-short} to ensure compatibility.
-Customers are responsible for Secured Cluster services upgrades required to maintain this version compatibility.
\ No newline at end of file
+To ensure optimal functionality and compatibility between Central and secured clusters, upgrade your secured clusters to the latest software version available. (The software version for Central is automatically updated.)
```

Comment:
Optional suggestion:

```suggestion
To ensure optimal functionality and compatibility between Central service and Secured Clusters, upgrade your secured clusters to the latest software version available. (The software version for Central is automatically updated.)
```

Reply:
secured clusters are not capitalized in the docs.